types/component/ChatList/ChannelImage.types.ts

```diff
@@ -2,18 +2,19 @@
 
 import {ImageStyle} from 'react-native-fast-image';
 import {StyleProp} from 'react-native';
+import {BetterSocialChannelType} from '../../database/schema/ChannelList.types';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `BetterSocialChannelType` is a good change as it replaces hardcoded string literals with an enum, which improves code maintainability and reduces the chance of typos.



<!-- This is an auto-generated comment by OSS CodeRabbit -->